src/2_RTB/Glossario/terms.csv

```diff
@@ -21,3 +21,27 @@ PoC; Acronimo di Proof of Concept, indica un prototipo iniziale che verifica la
+Sprint review; Evento fondamentale all'interno della metodologia Agile, specialmente nel contesto del framework Scrum. Si tratta di una riunione che si tiene alla fine di ogni sprint (iterazione di sviluppo) durante la quale il team di sviluppo presenta il lavoro completato durante lo sprint agli stakeholder, al product owner e ad altri membri interessati. Durante la Sprint Review, vengono esaminati i risultati ottenuti e si discute il progresso verso gli obiettivi dell'iterazione. È un'opportunità per raccogliere feedback, valutare il prodotto e adattare le priorità in vista degli sprint successivi.
+Materialized views; Rappresentano copie fisiche di dati risultanti da una query di selezione predefinita. A differenza delle viste standard, che eseguono la query ogni volta che vengono richiamate, le viste materializzate memorizzano fisicamente i dati calcolati dalla query in un'apposita struttura di archiviazione. Ciò consente un accesso rapido ai dati pre-calcolati, migliorando le prestazioni delle query che coinvolgono operazioni complesse o intensive.
+Retention policy; In Kafka, una "retention policy" (politica di conservazione) si riferisce alle regole e alle impostazioni che determinano per quanto tempo i messaggi devono essere conservati all'interno di un topic. Questa politica definisce la durata massima per cui i dati saranno mantenuti prima di essere eliminati automaticamente dal sistema. La retention policy è cruciale per gestire lo spazio di archiviazione e garantire che Kafka mantenga solo i dati necessari per il periodo di tempo desiderato. 
+Batch; In Kafka, un "batch" si riferisce a un insieme o a un gruppo di messaggi che vengono trattati e inviati insieme in una singola operazione. L'utilizzo di batch è un'ottimizzazione di prestazioni che consente di aggregare più messaggi in una singola unità di lavoro, riducendo il carico operativo e migliorando l'efficienza della trasmissione di dati all'interno di un sistema Kafka.
+Broker; In Kafka, un "broker" è un nodo individuale all'interno di un cluster Kafka responsabile della gestione e dell'elaborazione dei dati. Ogni broker svolge il ruolo di server Kafka e contribuisce al mantenimento e al funzionamento del sistema.
+Topic; In Apache Kafka, un "topic" è una categoria o una suddivisione logica dei dati che consente di organizzare e suddividere i flussi di messaggi. I topic sono fondamentali nel modello di pubblicazione/sottoscrizione di Kafka, dove i produttori inviano messaggi a specifici topic e i consumatori si sottoscrivono a tali topic per ricevere i messaggi di loro interesse.
```

Comment:
```suggestion
Topic; In Apache Kafka, un "topic" è una categoria o una suddivisione logica dei dati che consente di organizzare e suddividere i flussi di messaggi. I topic sono fondamentali nel modello di pubblicazione/sottoscrizione di Kafka, dove i produttori inviano messaggi a specifici topic e i consumatori si sottoscrivono a tali topic per ricevere i messaggi di loro interesse. I topic sono scalabili e flessibili, consentendo l'implementazione di soluzioni distribuite e la gestione efficiente di grandi volumi di dati in tempo reale.
```

Reply:
Secondo me non serve come aggiunta. Anche perché non sono una cosa fatta da noi. Ci sta così com'è, ovvero una breve descrizione di come funzionano in Kafka. Poi rischiamo anche di usare termini sbagliati.